docs/sources/enterprise/vault.md

```diff
@@ -0,0 +1,136 @@
++++
+title = "Vault"
+description = ""
+keywords = ["grafana", "vault", "configuration"]
+type = "docs"
+[menu.docs]
+parent = "enterprise"
+weight = 700
++++
+
+# Vault integration
+
+> Only available in Grafana Enterprise v7.1+.
+
+For organizations managing their secrets with [Hashicorp Vault](https://www.hashicorp.com/products/vault),
+Grafana integrates with Vault for [Configuration]({{< relref "../administration/configuration.md" >}})
+and [Provisioning]({{< relref "../administration/provisioning.md" >}}).
+
+> Note: If you have Grafana set up for [high availability]({{< relref "../tutorials/ha_setup.md" >}}) we advise not to use dynamic secrets for provisioning files.
+> Each Grafana instance is responsible for renewing its own leases. Your data sources' leases might expire when one of your Grafana servers shuts down.
+
+## Configuration
+
+Before using Vault you need to activate it by providing a URL, authentication method (currently only token),
+and a token for your Vault service. Grafana will automatically renew the service token if it is renewable and
+set up with a limited lifetime.
+
+If you're using short-lived leases, you can also configure how often Grafana should renew the lease and for
+how long. We recommend keeping the defaults here unless you run into problems.
+
+```ini
+[keystore.vault]
+# Location of the Vault server
+;url =
+# Vault's namespace if using Vault with multi-tenancy
+;namespace =
+# Method for authenticating towards Vault. Vault is inactive if this option is not set
+# Possible values: token
+;auth_method =
+# Secret token to connect to Vault when auth_method is token
+;token =
+# Time between checking if there are any secrets which needs to be renewed.
+;lease_renewal_interval = 5m
+# Time until expiration for tokens which are renewed. Should have a value higher than lease_renewal_interval
+;lease_renewal_expires_within = 15m
+# New duration for renewed tokens. Vault may be configured to ignore this value and impose a stricter limit.
+;lease_renewal_increment = 1h
+```
+
+Example for `vault server -dev`:
+
+```ini
+[keystore.vault]
+url = http://127.0.0.1:8200 # HTTP should only be used for local testing
+auth_method = token
+token = s.sAZLyI0r7sFLMPq6MWtoOhAN # replace with your key
+```
+
+
+## Using the Vault expander
+
+When you've configured Vault you'll need to set the configuration or provisioning files you wish to
+use Vault. Vault configuration is an extension of configuration's [variable expansion]({{< relref "../administration/configuration.md#variable-expansion" >}}) and follows the
+`$__vault{<argument>}` syntax.
+
+The argument to Vault consists of three parts separated by a colon: the first part specifies which secrets engine
```

Comment:
Make the three parts a bulleted list, each part with a separate bullet and treated like a separate sentence.